easybuild/easyconfigs/n/NSS/NSS-3.65-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'MakeCp'
+
+name = 'NSS'
+version = '3.65'
```

Comment:
There is a 3.69 available (https://ftp.mozilla.org/pub/security/nss/releases/NSS_3_69_RTM/src/). Is there a reason for sticking to 3.65?

Reply:
Homepage still listed 3.65 as the stable release but i think the homepage was just oudated.